### PR TITLE
Records api path refactor done

### PR DESCRIPTION
**Pull request by TeroOtavia**
Records api path refactor done

Resolves: #6678

### CODE REVIEW

**Review by avukkonen — APPROVED** [review]

File: muikku-core-plugins/src/main/resources/META-INF/resources/swagger.yaml [hunk 1]
```diff
@@ -12770,7 +12771,7 @@ components:
 
     RecordWorkspaceActivityInfo:
       type: object
-      description: RecordsWorkspaceActivityInfo object
+      description: Record workspace activity info with line and category, default line and activities
       required:
         - lineName
         - lineCategory
```
Comment (avukkonen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/swagger.yaml:12777]: These are not "lines", these are studyPrograms

**Review by avukkonen — CHANGES_REQUESTED** [review]

File: muikku-core-plugins/src/main/resources/META-INF/resources/swagger.yaml (hunk 1 above)
Comment (TeroOtavia) [reply muikku-core-plugins/src/main/resources/META-INF/resources/swagger.yaml:12777]: Rest speaks about lines.

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/main-function/records/index.ts [hunk 2]
```diff
@@ -8,13 +8,13 @@ import {
   TranscriptOfRecordLocationType,
   CurrentStudentUserAndWorkspaceStatusType,
   CurrentRecordType,
-  RecordWorkspaceActivityInfo,
   RecordWorkspaceActivityByLine,
```
Comment (avukkonen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/main-function/records/index.ts:11]: Out of scope, but this needs to be addressed somewhere. These are not lines, they are studyPrograms.

File: muikku-core-plugins/src/main/resources/META-INF/resources/swagger.yaml (hunk 1 above)
Comment (avukkonen) [reply muikku-core-plugins/src/main/resources/META-INF/resources/swagger.yaml:12858]: I'll make an issue about this. Carry on.

**Review by avukkonen — APPROVED** [review]

**Pull request merged**